### PR TITLE
Fix/arm/bitcoind sync

### PR DESCRIPTION
**Pull request by arminsabouri**

### CODE REVIEW

**Review by scottmillner — APPROVED** [review]

File: crates/btc-wallet/src/bitcoind.rs [hunk 1]
```diff
@@ -81,19 +81,14 @@ impl BitcoindClient {
     }
 
     pub async fn is_synced(&self) -> Result<bool, BitcoindError> {
-        match self.rpc.get_index_info().map_err(BitcoindError::BlockIndexStatusFailed) {
```
Comment (rwlockbg) [inline crates/btc-wallet/src/bitcoind.rs:84]: Any reason why this used to work and now it does not anymore ?

Comment (rwlockbg) [reply crates/btc-wallet/src/bitcoind.rs:84]: Can we maybe still keep the old code and if it fails, we try to call `get_blockchain_info` ? 
Or is the latter guaranteed to work with all bitcoinds ?

Comment (arminsabouri) [reply crates/btc-wallet/src/bitcoind.rs:84]: The old code was giving false positives. I think block index is available before IBD is finished

Comment (rwlockbg) [reply crates/btc-wallet/src/bitcoind.rs:84]: fair enough. Approved

**Review by rwlockbg — APPROVED** [review]

**Pull request merged**